### ISSUE TITLE
Allow constructing IVec2 in graph

### ISSUE THREAD

**Issue by 0HyperCube** [GraphiteEditor/Graphite#1919]
The artboard node takes an IVec2 as the dimensions and position. Currently it is impossible to construct this in the graph.

A node taking in two f64s and constructing an IVec2 should be added.

**Comment by Keavon** [comment]
We should consider reusing the Vector2 node and letting the type system support casting from an IVec2 to a DVec2.

**Comment by aybdee** [comment]
I want to try doing this to get a better feel for how graphene works, how do you suggest i go about it ?

**Comment by Keavon** [comment]
@TrueDoctor is this okay to work on right now or should it wait until your refactor in #1942 is done?

**Comment by Keavon** [comment]
In that case, @aybdee, you can just reference any of the value nodes. But if you want to wait like a week, the process of creating nodes will become much easier with that refactor mentioned above.

**Comment by aybdee** [comment]
i think i'd wait then 
i'll try some other stuff for now

**Comment by Keavon** [comment]
Sounds great. There are a lot of good options in `#code-todo-list` on Discord, that's where most of the good beginner issues are at.

**Comment by lizard-demon** [comment]
Is it still impossible to create an Ivec2?

**Comment by Keavon** [comment]
Yes, but automatic type conversion is right around the corner and it will be supported then.
